lib/trino-filesystem-cache-alluxio/src/main/java/io/trino/filesystem/alluxio/AlluxioFileSystemCacheConfig.java

```diff
@@ -40,12 +40,18 @@ public class AlluxioFileSystemCacheConfig
     static final String CACHE_MAX_SIZES = "fs.cache.max-sizes";
     static final String CACHE_MAX_PERCENTAGES = "fs.cache.max-disk-usage-percentages";
 
+    static final String CACHE_ON_WORKER = "fs.cache.coordinator-enabled";
+    static final String CACHE_ON_COORDINATOR = "fs.cache.worker-enabled";
```

Comment:
I'm not sure if the cache-only-on-worker permutation of configurations is practically useful. If someone can put local disks for caching on workers, they can do it on a coordinator too.
It's also a bit confusing to have the boolean `fs.cache.enabled` in FileSystemConfig
I would suggest an Enum config `fs.cache.mode=DISABLED/COORDINATOR_ONLY/ALL_NODES` and deprecate the existing `fs.cache.enabled`

Reply:
Makes sense.